WalletWasabi/BitcoinCore/Configuration/Whitening/WhiteEntry.cs

```diff
@@ -8,7 +8,7 @@ namespace WalletWasabi.BitcoinCore.Configuration.Whitening
 	public abstract class WhiteEntry
 	{
 		public string Permissions { get; private set; } = "";
-		public EndPoint EndPoint { get; private set; } = null;
+		public EndPoint? EndPoint { get; private set; } = null;
```

Comment:
Here instead of making this nullable, what makes no sense at all because this should never be null, I think you can simply make it:

```c#
public EndPoint EndPoint { get; init; }
```

And create a constructor like this one:
```c#
public WhiteEntry(EndPoint endPoint, string permissions)
{
	EndPoint = endPoint;
	Permissions = permissions;
}
```

In this way we have less nulls. Remember we don't want to have nullable types.

Reply:
Done

Reply:
`EndPoint` is still Nullable, isn't it?

Reply:
Yes it is, I did what Lucas suggested but there was an error so then I reverted that.

Reply:
This would be a bigger change. I tried the naive solutions but it is not possible because

`		public static bool TryParse<T>(string value, Network network, out T whiteEntry) where T : WhiteEntry, new()`

WhiteEntry can have only an empty constructor. I am merging, this can be improved in another PR.